librpc/Rpc.cpp

```diff
@@ -107,6 +107,19 @@ std::string Rpc::getPbftView(int _groupID)
         RPC_LOG(INFO) << LOG_BADGE("getPbftView") << LOG_DESC("request")
                       << LOG_KV("groupID", _groupID);
 
+        auto ledgerParam = ledgerManager()->getParamByGroupId(_groupID);
+        if (!ledgerParam)
+        {
+            BOOST_THROW_EXCEPTION(
+                JsonRpcException(RPCExceptionType::GroupID, RPCMsg[RPCExceptionType::GroupID]));
+        }
+        auto consensusParam = ledgerParam->mutableConsensusParam();
+        std::string consensusType = consensusParam.consensusType;
+        if (consensusType == "raft")
```

Comment:
If we have so many other consensus algorithms ?
I think it's better to use 'consensusType != 'pbft'